internal/node/test/BUILD.bazel

```diff
@@ -68,6 +68,12 @@ nodejs_binary(
     entry_point = ":module-name.js",
 )
 
+nodejs_binary(
+    name = "workers",
+    data = [":worker-main.js", "worker-thread.js"],
+    entry_point = ":worker-main.js",
+)
+
```

Comment:
I've extended that test a bit so it does a `require('lib')` where the lib is a bazel defined module, if the patcher hasn't been run then doing this will throw